src/views/Dissolutions/ReviewConfirmDissolution.vue

```diff
@@ -668,4 +766,22 @@ export default class ReviewConfirmDissolution extends Mixins(DateMixin) {
     }
   }
 }
+
+// styles specific to resolution summary section
```

Comment:
It's fair to have this here atm, but this should go into a sub-component if possible.

It's not possible to move the other specific CSS into their components since those are shared components.

Reply:
will move into sub-component when I create the resolution summary component.